tests/strategies/external/test_pv.py

```diff
@@ -132,6 +150,27 @@ def test_offer_aggregator(external_pv):
         )
         assert_bid_offer_aggregator_commands_return_value(return_value, True)
 
+    @staticmethod
+    def test_offer_aggregator_places_future_offer(external_pv, future_markets):
```

Comment:
I see tests for placing bids/offers and deleting them. How about updating of bids/offers? Isn't it needed?

Reply:
An sort-of update can be achieved by setting `replace_existing=True`. With this, the existing order is deleted and the updated values are posted in a new order. We need to do this in order to broadcast to all listeners, that the order was updated, only with the existing implementation, two events are broadcasted: DELETE_OFFER and NEW_OFFER. 
I hope it makes sense.